### DISCUSSION TITLE
什麼事GitHub Flow？

### DISCUSSION THREAD

**Discussion by liangyinglly** [1101-GS4719-C/Python-Community-GS4719-C#659] [General]
最近在做期末專題 很多人會需要一起寫一支程式
才意識到 work flow 真的太重要了 讓大家工作起來更有效率
在此分享什麼是GitHub Flow
大致上是經過1.建立分支2.新增提交3.開啟 Pull Request4.討論與檢閱你的程式碼5.部署6.合併
更多詳細資訊請看：https://medium.com/@trylovetom/%E8%AE%93%E6%88%91%E5%80%91%E4%BE%86%E4%BA%86%E8%A7%A3-github-flow-%E5%90%A7-4144caf1f1bf

**Comment by HsuChen88** [comment]
我最近也是在做期末
建立了一個儲存庫來共編
但因為只有兩個人沒有開分支
（其實是因為我不太會用）
但謝謝你提供的資訊
希望未來可以再做更大的專案

**Comment by gnsJhenJie** [comment]
自動部署真的很方便, 尤其是部署需要很多步驟的狀況下, 如果手動部署可能每個人做的方法或順序都不一樣, 用GitHub Flow可以確保每次的部署流程都相同。另外, GitLab也有提供類似的服務叫做GitLab CI/CD。

**Reply by liangyinglly** [reply]
謝謝分享 馬上來去研究一下

**Comment by Lynn12378** [comment]
好像在哪看過你XD
另外github的workflow還可以在創建網站的時候設定自動部署，超方便。
